src/main/resources/templates/macros/array.js.vm

```diff
@@ -0,0 +1,2 @@
+#macro(stringArray $array) [#foreach($element in $array) "$element", #end] #end
```

Comment:
name of the file tells developer that there is some JavaScript code here - there is no